packages/expo/package.json

```diff
@@ -52,16 +53,13 @@
     "@types/react": "*",
     "@types/react-dom": "*",
     "eslint-config-custom": "*",
-    "expo-auth-session": "^5.4.0",
-    "expo-web-browser": "^12.8.2",
```

Comment:
You'll still need them in `devDeps` to develop the package locally, otherwise you'll have TS errors in https://github.com/clerk/javascript/blob/71663c568926b1d60b97aa7ccc5298d05b618af2/packages/expo/src/useOAuth.ts

Reply:
We will keep them as devDeps to support cases of package managers not installing peer deps by default.